wolfcrypt/src/ecc.c

```diff
@@ -5879,6 +5882,7 @@ int wc_ecc_sign_hash(const byte* in, word32 inlen, byte* out, word32 *outlen,
     (void)inlen;
     (void)s;
     (void)r;
+    (void)err;
 #endif /* WOLF_CRYPTO_CB_ONLY_ECC */
 
 }
```

Comment:
Can we hit this without a return?

Reply:
Can't get to these lines but they are here to make the variables look used.
(Took me a little while to see this!)

Reply:
I'll try to rework this to make sense!

Reply:
Nope but fixed to be clearer.